README.md

```diff
@@ -8,7 +8,7 @@
 [![GitHub Workflow Status (branch)](https://img.shields.io/github/workflow/status/phylum-dev/phylum-ci/Test/main?label=Test&logo=GitHub)](https://github.com/phylum-dev/phylum-ci/actions/workflows/test.yml)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](./CODE_OF_CONDUCT.md)
 
-Python package for handling CI and other integrations
+Utilities for handling Phylum integrations
```

Comment:
It is both...and I also struggled with the language to use here. The goal was to be succinct, clear, and light (if possible...think "levity"). The same description is also used in the Python package description. It will be changed there as well.